src/plugin.c

```diff
@@ -151,6 +152,11 @@ ncclResult_t ncclIbInit(ncclDebugLogger_t logFunction) {
   if (ncclNIbDevs == -1) {
     pthread_mutex_lock(&ncclIbLock);
     wrap_ibv_fork_init();
+
+    if (ncclParamIbPciRelaxedOrdering() & !IBV_ACCESS_RELAXED_ORDERING) {
```

Comment:
is it supposed to be logical AND here? (not binary)

Reply:
i think binary would work. but logical is correct. fixed